lib/Scvmm/miq_scvmm_parse_powershell.rb

```diff
@@ -0,0 +1,11 @@
+
+class MiqScvmmParsePowershell
```

Comment:
There is already an MiqPowershell class.  This stdout handling thing, as well as the stderr parsing thing from @bronaghs's scvmm parser code seem like they belong there instead.  Then you don't need a special class here.